### PR TITLE
[stable/mongodb] updates to support alternate k8s scheduler

### PR DESCRIPTION
**Pull request by satchpx**
Signed-off-by: Sathya Balakrishnan <sathya@portworx.com>

#### What this PR does / why we need it:
Adds capability to specify an alternate k8s scheduler

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

File: stable/mongodb/values.yaml [hunk 1]
```diff
@@ -144,6 +143,11 @@ podAnnotations: {}
 # Additional pod labels to apply
 podLabels: {}
 
+## Use an alternate scheduler, e.g. "stork".
+## ref: https://kubernetes.io/docs/tasks/administer-cluster/configure-multiple-schedulers/
+##
+# schedulerName:
```
Comment (juan131) [inline stable/mongodb/values.yaml:149]: Apply the same changes in the `values-production.yaml` too

**Review by juan131 — CHANGES_REQUESTED** [review]
Thanks for the contribution! Please review my comments

File: stable/mongodb/values.yaml (hunk 1 above)
Comment (satchpx) [reply stable/mongodb/values.yaml:149]: Thanks for catching it. Update done

**Review by juan131 — APPROVED** [review]
/ok-to-test
/lgtm

**Pull request merged**